docs/user-guide/install-zowe-client-side-components.md

```diff
@@ -0,0 +1,21 @@
+# Installing Zowe client-side components
+
+Prepare yourself to install Zowe client-side components.
```

Comment:
How about:
Review this topic to prepare to install, configure, and deploy Zowe client-side components.